src/app/core/tools/InteractionTool.ts

```diff
@@ -10,6 +10,7 @@ import {IOObject} from "core/models";
 import {DefaultTool} from "./DefaultTool";
 
 import {EventHandler} from "./EventHandler";
+import {WiringTool} from "./WiringTool";
```

Comment:
So in general it's bad practice to have a tool import some other tool. In theory each tool should be completely independent and not have to explicitly depend on another one.

I'll mention how to fix this below